CHANGELOG.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "version": "0.1.0",
```

Comment:
sup w/ these duplicate entries?

Reply:
Thanks, addressed in https://github.com/0xProject/quote-server/pull/5/commits/598b8f91a808030f5ff59177ad11a1a328169b55